### PR TITLE
change command 'install' to 'apply'

### PR DESCRIPTION
**Pull request by lfbdev**
# Summary

change dtm command from 'install' to 'apply'

## Description

Describe what this PR does and what problems it tries to solve in a few sentences.

### Related Issues

#48

### CODE REVIEW

File: cmd/devstream/install.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var installCMD = &cobra.Command{
-	Use:   "install",
+	Use:   "apply",
 	Short: "Install tools defined in config file",
 	Long:  `Install tools defined in config file`,
```
Comment (IronCore864) [inline cmd/devstream/install.go:18]: For the short and long descriptions, I think we need a better one for the short and long descriptions for the short and long descriptions.

Example:

Short: Creates or updates DevOps tools according to DevStream configuration file.

Long: Creates or updates DevOps tools according to DevStream configuration file. DevStream will generate and execute a new plan based on the config file and the state file by default.

**Review by IronCore864 — CHANGES_REQUESTED** [review]

File: cmd/devstream/install.go (hunk 1 above)
Comment (IronCore864) [reply cmd/devstream/install.go:18]: Please also make sure "install" is changed to "apply" in all places, such as logging, etc. A grep might help.

Comment (lfbdev) [reply cmd/devstream/install.go:18]: > Please also make sure "install" is changed to "apply" in all places, such as logging, etc. A grep might help.

these logs may not be modified:
image

Comment (lfbdev) [reply cmd/devstream/install.go]: > For the short and long descriptions, I think we need a better one for the short and long descriptions for the short and long descriptions.
> 
> Example:
> 
> Short: Creates or updates DevOps tools according to DevStream configuration file.
> 
> Long: Creates or updates DevOps tools according to DevStream configuration file. DevStream will generate and execute a new plan based on the config file and the state file by default.

modified

**Review by daniel-hutao — APPROVED** [review]
/lgtm

**Review by IronCore864 — APPROVED** [review]

**Pull request merged**